test/unit/docstrings_test.py

```diff
@@ -0,0 +1,99 @@
+import inspect
+import pkgutil
+import pytest
+import re
+
+import kiwi
+
+
+def members(module, predicate=None):
+    """Yields all functions of a module if predicate=None
+
+    :param module: the given module to search for members
+    :type module: :class:`module`
+    :param predicate: a function with expects an object and returns True or False
+                      see :func:`inspect.isfunction` as an example
+    :type prediate: :class:`function`
+    """
+    predicate = inspect.isfunction if predicate is None else predicate
+    for _, func, in inspect.getmembers(module, predicate):
+        if func.__module__ == module.__name__:
+            yield func
+
+
+def getmodules(module, predicate=None):
+    """Yields recursively sub module names from a given module
+
+    :param module: the given module to search for members
+    :type module: :class:`module`
+    :param predicate: a function with expects an object and returns True or False
+                      see :func:`inspect.isfunction` as an example
+    :type prediate: :class:`function`
+    """
+    prefix = module.__name__ + "."
+    for importer, modname, ispkg in pkgutil.iter_modules(module.__path__, prefix):
+        module = importer.find_module(modname).load_module(modname)
+        yield from members(module, predicate)
```

Comment:
I don't think the order of tests can be influenced, and to be honest the order should not serve as a "fix". doing so will strike back sooner or later and at that time it will be very hard to remember that the order of the tests was significant for success or failure. I'm sorry